---
title: Research
layout: default
---
<!-- Featured -->
<div id="featured">
	<div class="container">
		<header>
			<h2>Research</h2>
		</header>
		<p>Research in the Hohenlohe Lab addresses basic questions in evolutionary genetics and genomics from both theoretical 
		and empirical perspectives, and also applies genomic tools to conservation of threatened species.</p>

	</div>
</div>

<!-- Main -->
	<div id="main">
		<div id="content" class="container">

			<div class="row">
				<section class="6u">
					<a href="#" class="image full"><img src="/assets/images/Ed_Kosmicki.png" alt="RADseq, photo: Ed Krosmicki"></a>
					<header>
						<h2>Population Genomics</h2>
					</header>
					<p>Genomic tools have revolutionized biology and our ability to understand genetic processes in natural populations.  Our lab
					studies a wide range of questions in population genomics, including the genetics of adaptation, population structure and 
					phylogeography, disease resistance, the genetic basis of complex phenotypes, and demographic reconstruction.  We use a common 
					set of molecular techniques such as Restriction site-Associated DNA sequencing (RADseq) and whole-genome sequencing, 
					combined with computational and bioinformatic tools, in species such as pygmy rabbits, threespine stickleback, trout, 
					tamarisk leaf beetles, capybaras, wolves, Channel Island foxes, Gobi bears, and more.</p>
				</section>
				<section class="6u">
					<a href="#" class="image full"><img src="/assets/images/Teleganscropped.png" alt="Conservation Genomics"></a>
					<header>
						<h2>Conservation</h2>
					</header>
					<p>Evolutionary genomic approaches have powerful applications to conservation of species and ecosystems.  
					We use genomic tools to develop large sets of genetic markers, assess phylogeographic structure, detect 
					hybridization and introgression, and understand the genetic basis of wildlife disease.  For example we are collaborating on a genomic 
					study of the evolution and epidemiology of <a href="http://www.nsf.gov/discoveries/disc_summ.jsp?cntn_id=129508">Tasmanian devil facial tumor disease</a>, 
					a unique transmissible cancer, that poses a serious threat of extinction for this iconic species.  Our goal is to 
					identify genetic variation in devil populations associated with disease progression and inform species management. 
					Other conservation projects address the remarkably high incidence of cancer in one population of island foxes, 
					hybridization and genetic rescue in Columbia Basin pygmy rabbits, and genetic monitoring of harvested Idaho gray wolves.</p>
				</section>
			</div>

			<div class="row">
				<section class="6u">
					<a href="#" class="image full"><img src="/assets/images/yeastplatecropped.png" alt="Experimental Population Genomics"></a>
					<header>
						<h2>Experimental Evolution</h2>
					</header>
					<p>The explosion of population genomic data from non-model organisms, made possible by techniques like RADseq, 
					has outstripped our understanding of genomic evolution and our ability to make sense of the data.  To bridge this gap, 
					we are combining experimental evolution in yeast with next-generation sequencing. Here we can control population 
					sizes, migration rates, strengths of selection, recombination rates, and amount and structure of standing genetic 
					variation in replicated experiments, and observe the results of evolution at the genomic sequence level.  
					The goal is to improve our powers of inference about natural populations from population genomic data.</p>
				</section>
				<section class="6u">
					<a href="#" class="image full"><img src="/assets/images/Fig7cropped.png" alt="Evolutionary Theory"></a>
					<header>
						<h2>Evolutionary Theory</h2>
					</header>
					<p>Members of the lab are involved in several projects developing novel theory, analytical tools, or 
					simulation approaches to understand evolutionary processes.  For example, we are studying the relationship between 
					genetic regulatory networks and the structure of variation in complex multivariate phenotypes (i.e. the M and G 
					matrices of quantitative genetics).  We have also 
					developed a novel method for estimating the dimensionality of evolution that is widely applicable to mate 
					choice and reproductive isolation, local adaptation, host-parasite coevolution, and other scenarios.</p>
				</section>
			</div>
			<br><br>
			
			<!-- <header><h2>Collaborators</h2></header>
			<ul><a href="http://www.cfc.umt.edu/personnel/details.php?ID=873">Fred Allendorf, University of Montana</a></ul>
			<ul><a href="http://wp.natsci.colostate.edu/funklab/">Chris Funk, Colorado State University</a></ul>
			<ul><a href="http://www.utas.edu.au/zoology/people/rodrigo-hamede">Rodrigo Hamede, University of Tasmania</a></ul>
			<ul><a href="http://www.utas.edu.au/zoology/people/menna-jones">Menna Jones, University of Tasmania</a></ul>
			<ul><a href="http://flbs.umt.edu/people/luikart3422/?id=3422">Gordon Luikart, University of Montana</a></ul>
			<ul><a href="https://www.griffith.edu.au/environment-planning-architecture/griffith-school-environment/staff/professor-hamish-mccallum">Hamish McCallum, Griffith University</a></ul>
			<ul><a href="http://www.vet.cam.ac.uk/directory/murchison">Elizabeth Murchison, Cambridge University</a></ul>
			<ul><a href="http://public.wsu.edu/~storfer/">Andrew Storfer, Washington State University</a></ul>
			-->
			<br>
			<!--research support... need to resize images!
			<header><h2>Research Support</h2></header>
			<ul><a href="http://www.nsf.gov/" class="image full"><img src="/assets/images/nsf4c.jpg" alt="NSF"></a></ul>
			<ul></ul>
			<ul></ul>
			<ul></ul>
			<ul></ul>-->

		</div>
	</div> 

